### ISSUE TITLE
All the starting "l" in host address will be removed unconditionally

### ISSUE THREAD

**Issue by charlesnieh** [pcman-bbs/pcmanx#84]
For example, trying to connect to **labc**.com will lead to a connection to **abc**.com.

It only works to add a backslash before l, i.e. **\labc**.com.

**Comment by Mushiyo** [comment]
This seems related to #69.

**Comment by holishing** [comment]
This bug has been fix in #69 and merged into master branch. 

You can report this problem to bug system of your distribution (Debian BTS, Ubuntu Launchpad, RedHat Bugzilla ... etc) to add related patch. (ps. FreeBSD ports has been applied)

Since there's no new release after #69 merged, I'm not sure if it is ok to add fix release of version 1.3, because I'm not the main maintainer.

**Comment by holishing** [comment]
If we want to discuss about release procedure, we should open new issue, close this issue.

**Comment by kidwm** [comment]
@holishing we can bump to 2.0 after GTK3 porting, may you help to check this?

**Comment by holishing** [comment]
I can help checking if basic features works and if builds can be passed in different unix-like platform, I cannot help further works like programming or advanced debugging.
